cognite/client/utils/_client_config.py

```diff
@@ -8,6 +8,8 @@
 from cognite.client import utils
 from cognite.client.exceptions import CogniteAPIKeyError
 
+_DEFAULT_API_SUBVERSION = "V20210323"
```

Comment:
this is a bad place for this to live long term, but i wasn't sure where to put it

Reply:
does this mean using the sdk will lock in api bugs fixed after 2 days ago?

Reply:
we can change the default api version when creating new sdk releases. And generally bugfixes will not trigger a new API version. It's more: the sdk will lock into a version of the API so it does not suddenly break if we make a breaking change to the API

Reply:
maybe we should make it live next to the version variable so you notice it more?